stories/Input.stories.tsx

```diff
@@ -6,6 +6,74 @@ import { Input, Form, Button } from '../src';
 
 const stories = storiesOf('Input', module);
 
+const form = (disableErrorFromComponents: boolean) => (
+  () => {
+    const [firstName, setFirstName] = useState<string>('');
+    const [middleName, setMiddleName] = useState<string>('');
+    const [lastName, setLastName] = useState<string>('');
+    const [firstNameError, setFirstNameError] = useState<string>('');
+    const [middleNameError, setMiddleNameError] = useState<string>('');
+    const [lastNameError, setLastNameError] = useState<string>('');
+
+    const onSubmit = (e: SyntheticEvent<HTMLFormElement>) => {
+      e.preventDefault();
+      setFirstNameError(firstName.length === 0 ? 'You must enter a first name' : null);
+      setMiddleNameError(middleName.length === 0 ? 'You must enter a middle name' : null);
+      setLastNameError(lastName.length === 0 ? 'You must enter a last name' : null);
+    };
+
+    return (
+      <Form onSubmit={onSubmit} disableErrorFromComponents={disableErrorFromComponents}>
+        <Input
+          className="nhsuk-u-margin-bottom-3"
+          autoFocus
```

Comment:
The `autoFocus` prop doesn't exist on the `<Input>` component, so it won't have any effect and may trigger a React warning